data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempt) VALUES ('Agumon','2020-02-03',true,10.23,0);
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempt) VALUES ('Gabumon','2018-11-15',true,8,0);
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempt) VALUES ('Pikachu','2021-01-07',false,15.04,0);
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempt) VALUES ('Devimon','2017-05-12',true,11,0);
```

Comment:
- [ ] As per the data mentioned in the project requirements, the escape attempt property is 0 only for Agumon, while for the others it should be 2, 1, and 5 please update this data